articles/tokens/refresh-token.md

```diff
@@ -1,152 +1,173 @@
 ---
-description: A refresh token allows an application to request Auth0 to issue a new id_token directly, without needing to re-authenticate the user.
+title: Refresh Token
+description: What is a Refresh Token and how you can use it.
 ---
 
-# Refresh Tokens
+# Refresh Token
```

Comment:
Let's add a section talking about the "refresh story" for SPA (The `renewAuth` method in the API, all documentation warnings apply https://github.com/auth0/auth0.js#api)